frontend/src/state/supportedFeatures.ts

```diff
@@ -134,6 +138,9 @@ class SupportedFeatures {
   @computed get rpcnSecretsApi(): boolean {
     return isSupported(Feature.SecretService);
   }
+  @computed get schemaRegistryACLService(): boolean {
```

Comment:
I think the convention established so far is to do
```suggestion
  @computed get schemaRegistryACLApi(): boolean {
```